### ISSUE TITLE
Incorrect argument passed to `system2` call in `invoke_cargo`

### ISSUE THREAD

**Issue by Ilia-Kosenkov** [extendr/rextendr#111]
I was looking for a way to capture `cargo` output when the compilation errors when I found this:
https://github.com/extendr/rextendr/blob/942d0d3b749ee7c957a003631aa7e9658c3be24f/R/source.R#L246-L259
`stdout` passed to both `stdout` and `stderr`.
The function, however, has both `stdout` and `stderr` arguments
https://github.com/extendr/rextendr/blob/942d0d3b749ee7c957a003631aa7e9658c3be24f/R/source.R#L223-L224

The wild thing is the caller of `invoke_cargo` passes to `stderr` an invalid value of `stderr`, which is **not assigned** in the caller method.
https://github.com/extendr/rextendr/blob/942d0d3b749ee7c957a003631aa7e9658c3be24f/R/source.R#L164-L172

Basically, we have two bugs that cancel each other because of the lazy evaluation of function arguments in R...
